src/measure_theory/probability_mass_function/constructions.lean

```diff
@@ -25,77 +25,13 @@ and `filter` uses this to filter the support of a `pmf` and re-normalize the new
 
 -/
 
-namespace pmf
+universes u v
 
 noncomputable theory
-variables {α : Type*} {β : Type*} {γ : Type*}
+variables {α β γ : Type u}
```

Comment:
As discussed on Zulip, I don't think we want want to be throwing away this universe polymorphism. See for instance how we have `option.map`, which is universe polymorphic, in addition to providing the monad API on `option` which is not.